content/blog/pulumi-bloggers-oct-2019/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Four Infrastructure as Code Blog Posts You Should Read"
+authors: ["sophia-parafina"]
+tags: ["Pulumi","bloggers",]
+meta_image: "blog.png"
+meta_desc: "Continuous delivery of Pulumi apps with Octopus Deploy "
+date: "2019-10-24"
+---
+
+We are always excited when people join the Infrastructure as Code community and write about their experiences. Here are four new posts that we think are great!
+
+## CircleCI on Google Kubernetes Engine (GKE)
+
+[Angel Rivera](https://twitter.com/punkdata) shows how to set up CircleCI on GKE with a [step-by-step tutorial](https://circleci.com/blog/automate-releases-from-pipelines-using-infrastructure-as-code/). Angel demonstrates how to integrate Infrastructure as Code solutions, such as Pulumi, into CI/CD pipelines. The tutorial builds a container with a Pulumi project with a sample app written in Python that deploys Kubernetes. The tutorial shows how you can incorporate your infrastructure into your CI/CD pipeline. The code for the tutorial is available in [GitHub](https://github.com/datapunkz/orb-pulumi-gcp).
+
+## HATech Compares Pulumi to Terraform
+
+[Jon Hathway](https://twitter.com/hatechllc) of HATech published a white paper comparing Pulumi to Terraform and explained why HATech is evaluating Pulumi.
+
+> Take for example the teams that are trying to collaborate and drive reliably deployed and maintained product innovations. Is it better to have languages that support each individual team’s needs, or is it better to have one language to support the entire workflow/pipeline?
+>
+> How about this thought experiment: Imagine your developers only spoke English, your QA team only spoke French, and the operations team only spoke Spanish? It would be carnage!
+>
+>You’d never hire that way. So why, when choosing DevOps tools, do people find it acceptable to have different languages, DSLs that only key individuals understand?”
+
+Jon provides an in-depth review of Pulumi and includes the [code samples](https://github.com/hatech/demo-pulumi-python) he used on GitHub. Download the white paper from [HATech](https://hatech.io/pulumi-v-terraform-whitepaper-lp/).
+
+## VMWare VSphere and Pulumi
+
+![peek definition](peek-definition.jpg)
+
+[Cody De Arkland](https://twitter.com/Codydearkland) demonstrates how to create a VMWare VSphere Virtual Machine by adding a ResourcePool, DataStore, NetworkStore and the template with Pulumi. He also sets the configuration values such as DNS settings, default gateway, domain, and hostname all in code. All of this runs on the ESXi virtualization server and vCenter (which is why the Datacenter ID is needed for configuration). His example is available on [GitHub}(https://github.com/codyde/pulumi-vsphere-ts). We like Cody’s caveat that his post is not an endorsement but call to explore new tools. In his conclusion, he nails home the point why Infrastructure as Code is the future of DevOps.
```

Comment:
* I could be missing some context, but what is the template referred to here? `and the template with Pulumi.`
* The link is not rendering properly due to brackets `[GitHub}(https://github.com/codyde/pulumi-vsphere-ts)`
* `We like Cody’s caveat that his post is not an endorsement but call to explore new tools.` is a bit of a strange add in.. although I didnt have a link to the actual article so maybe its not strange?

Reply:
I added 'virtual machine' template
fixed link
Cody says the blog wasn't an endorsement but something to add to your tool box.